pom.xml

```diff
@@ -178,6 +178,14 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.2</version>
             </plugin>
+            <plugin>
```

Comment:
I think this part is not needed. 
This plugin is build with java 8

Reply:
ok, fixed